person.rb

```diff
@@ -1,20 +1,27 @@
 require 'securerandom'
+require_relative 'nameable'
 
-class Person
-  attr_accessor :name, :age, :parent_permission
+class Person < Nameable
```

Comment:
- [ ] Once more let's kindly make sure we can decorate the person class. As requested by the previous reviewer try to tests if your codes are working. Create a person object and try to tests if you can correct, capitalized and trim the name. I'm getting the error on the console below:
![Screenshot from 2023-11-08 20-05-30](https://github.com/Ochiengsteven/school-library-oop/assets/63087491/31d228fe-cfc7-4124-97d3-3a843ab80d5b)

Reply:
Hello @FrederickMih, I've added `test.rb` file that require `person.rb` and `nameable.rb` and if I run it everything is ok